### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by DanteAlonsoHT**
We've been working on the Milestone 3, we've considered the responsive design using Bootstrap classes, we've added the next sections for the project:

- _In the magazine and Editor’s pick_
- _Newspaper main sections_
- _Subscribe / Newsletter_

Finally, we've passed the linters successfully.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -153,6 +178,40 @@ a:hover {
   margin-top: 14px;
 }
 
+.gray-container {
+  min-height: 450px;
+  padding: 30px;
+}
+
+.newsletter-img2 {
+  position: absolute;
```
Comment (smcommits) [inline css/style.css]: You can use bootstrap class for this

File: css/style.css [hunk 2]
```diff
@@ -153,6 +178,40 @@ a:hover {
...
+  top: 8px;
+  left: 64px;
+}
+
+.newsletter-container {
+  justify-content: center;
```
Comment (smcommits) [inline css/style.css]: You can use bootstrap class fo this.

File: css/style.css [hunk 3]
```diff
@@ -188,14 +251,33 @@ a:hover {
...
+  }
+
+  .gray-container {
+    min-height: 302px;
+    max-height: 302px;
+    padding: 30px;
+  }
+
+  .newsletter-container {
+    justify-content: start;
```
Comment (smcommits) [inline css/style.css]: You can use bootstrap class for this.

File: css/style.css [hunk 4]
```diff
@@ -250,6 +332,10 @@ small {
   .footer-list-links li:last-child {
     border: none;
```
Comment (smcommits) [inline css/style.css]: You can use bootstrap class for this.

File: css/style.css [hunk 5]
```diff
@@ -267,3 +353,40 @@ input:focus {
 .summary {
   font-family: Arial, Helvetica, sans-serif;
 }
+
+.square {
+  position: relative;
```
Comment (smcommits) [inline css/style.css]: You can use bootstrap class for this.

**Review by smcommits — CHANGES_REQUESTED** [review]
# Status: Changes Required 
## Hello, @kensayo and @DanteAlonsoHT! You guys have done an excellent job. The amount of work you have put in to clone the page is highly impressive. From images to fonts, everything looks great. The website appears to be fantastic. 
To make this project even better, we need to address some issues. 

## Required Changes

[ ] **In Editor's Pick section, images should be link**
	
Editor's pick section
:-------------------------:
editor'spick-new

<hr>

- [x] **The input in Newsletter Section should be of type email**

The `<input type="email">` defines a field for an e-mail address.
The input value is automatically validated to ensure it is a properly formatted e-mail address.

Newsletter Section
:------------------------:
inputType-new

<hr> 

- [x] **You should bootstrap classes wherever possible**

In some places, you are creating css rules just apply the properties that can be easily applied with bootstrap. This defeats the purpose of using a framework. Please refer to the inline comments to understand more about this. 


# 

## Please request another review from your dashboard once you have implemented the changes. 

### :warning: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 2 code review left for this project.

### Please reach out to me on [Slack](https://microverse-students.slack.com/team/U017N7LLZ40) if you need any assistance.

File: css/style.css (hunk 4 above)
Comment (DanteAlonsoHT) [reply css/style.css]: Thanks, I didn't know how but now I learned something new :)

**Review by christianotieno — APPROVED** [review]
### Milestone Approved 🍾 
Hello @DanteAlonsoHT & @kensayo :wave:
##
This milestone has now met all the requirements needed for approval. 👏🏾 
Congratulations on the hard work done with the changes and overall, on this milestone. Good job 👍🏾 
I will go ahead and mark this as complete. Good luck with the next tasks 🥇 
##
### Happy Coding 💻 
#### Christian Otieno (CR)
**| [Twitter](https://twitter.com/iamchrisotieno) | [Github](https://github.com/christianotieno) | [LinkedIn](https://www.linkedin.com/in/christianotieno/) |**

**Pull request merged**